src/ServiceControl.Monitoring/Http/Diagrams/TimingsAggregator.cs

```diff
@@ -111,6 +64,7 @@ public class MonitoredEndpoint
         public string[] EndpointInstanceIds { get; set; }
         public MonitoredEndpointValues ProcessingTime { get; set; }
         public MonitoredEndpointValues CriticalTime { get; set; }
+        public MonitoredEndpointValues Retries { get; set; }
```

Comment:
I propose to make this a separate PR in which I will try to make `MonitoredEndpointValue` more generic.